src/Logging/test/Microsoft.Extensions.Logging.Tests.csproj

```diff
@@ -4,6 +4,11 @@
     <TargetFrameworks>$(DefaultNetCoreTargetFramework);net472</TargetFrameworks>
   </PropertyGroup>
 
+  <ItemGroup>
```

Comment:
@anurse I had to add this reference to be able to use `ISystemClock`, `SystemClock` & `TestClock`. This feels dirty and should be moved out of `Microsoft.Extensions.Caching.Abstractions.csproj` but not sure if 1) you agree and 2) this should be done via this PR.

Reply:
Removed